jina/drivers/rank.py

```diff
@@ -109,41 +118,6 @@ class CollectMatches2DocRankDriver(BaseRankDriver):
     def __init__(self, traversal_paths: Iterable[str] = ['m'], *args, **kwargs):
         super().__init__(traversal_paths=traversal_paths, *args, **kwargs)
 
-    def _apply_all(self, docs: Iterable['jina_pb2.Document'], context_doc: 'jina_pb2.Document', *args, **kwargs) -> None:
```

Comment:
why are we removing this, it does not behave like `Chunk2DocRankDriver`?. It has some differences

Reply:
ok my bad, yes, they have different loop structure

Reply:
I will put it back